test/perf/skipRepeats.js

```diff
@@ -29,11 +30,14 @@ suite
 	.add('most', function(deferred) {
 		runners.runMost(deferred, most.from(a).skipRepeats().reduce(sum, 0));
 	}, options)
-	.add('rx', function(deferred) {
+	.add('rx 4', function(deferred) {
 		runners.runRx(deferred, rx.Observable.fromArray(a).distinctUntilChanged().reduce(sum, 0));
 	}, options)
+  .add('rx 5', function(deferred) {
```

Comment:
I just noticed the indenting in the tests, themselves.  I think that's the only thing left.  Otherwise, this looks great.  I ran it locally (using `npm start`) and it all worked perfectly.